src/redux/books/book.js

```diff
@@ -0,0 +1,30 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), So please edit all action types to be something like this:
`const ADD_BOOK = bookstore/books/ADD_BOOK`

Reply:
@sumairq **Thanks for great feedback**